roles/memory/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: capture meminfo
+  slurp:
+    src: /proc/meminfo
```

Comment:
we'll need to add the ignore_errors if for some reason the file doesn't exist... 

What i'll recommend is check if file exists and then put these 2 in a block statement.

Reply:
done